src/Site_Command.php

```diff
@@ -251,16 +251,20 @@ function delete( $args, $assoc_args ) {
 			$blog = get_blog_details( trim( $assoc_args['slug'], '/' ) );
 		} else {
 			if ( empty( $args ) ) {
-				WP_CLI::error( "Need to specify a blog id." );
+				WP_CLI::error( 'Need to specify a blog id.' );
 			}
 
 			$blog_id = $args[0];
 
+			if ( is_main_site( $blog_id ) ) {
+				WP_CLI::error( 'You can\'t delete root blog.' );
```

Comment:
Should we explain why the root blog can't be deleted?

Reply:
I don't think that's necessary, users will do understand that WordPress doesn't allow deletion of root blog.

Reply:
Ok. Updated the language a bit in abae4141